sfdc/modules/bulk/tests/bulk_xml_query.bal

```diff
@@ -114,8 +114,18 @@ function queryXml() returns error? {
     foreach var i in 1 ..< maxIterations + 1 {
         var batchResult = baseClient->getBatchResult(queryJob, batchId);
         if (batchResult is xml) {
-            test:assertTrue((batchResult/<*>).length() == 4, msg = "Retrieving batch result failed.");
-            break;
+            if ((batchResult/<*>).length() == 4) {
```

Comment:
Is it possible to define two variables like this and do
1. int maxAssertRetries = 5
2. int delayBetweenReries = x
3. replace `i` with `currentRetry` to be meaningful.

Reply:
Resolved in https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/189/commits/164cb7658171869f079b5521e4442ef1b148054e